src/main/rx/rx.c

```diff
@@ -416,11 +417,21 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
 #endif
     {
         const uint8_t frameStatus = rxRuntimeConfig.rcFrameStatusFn(&rxRuntimeConfig);
-        if (frameStatus & RX_FRAME_COMPLETE) {
+        if (frameStatus & (RX_FRAME_COMPLETE | RX_FRAME_FAILSAFE | RX_FRAME_DROPPED)) {
             rxDataProcessingRequired = true;
             rxIsInFailsafeMode = (frameStatus & RX_FRAME_FAILSAFE) != 0;
-            rxSignalReceived = !rxIsInFailsafeMode;
+            rxSignalReceived = (frameStatus & RX_FRAME_COMPLETE) != 0;
             needRxSignalBefore = currentTimeUs + needRxSignalMaxDelayUs;
+
+            if (frameStatus & RX_FRAME_DROPPED) {
+            	// No (0%) signal
+            	setRssiUnfiltered(0, RSSI_SOURCE_FRAME_ERRORS);
+            } else {
+            	// Valid (100%) signal
+            	setRssiUnfiltered(RSSI_MAX_VALUE, RSSI_SOURCE_FRAME_ERRORS);
+            }
+        } else if (cmpTimeUs(currentTimeUs, rxNextUpdateAtUs) > 0) {
```

Comment:
This isn't required here, since it's already done in https://github.com/SteveCEvans/betaflight/blob/04a11c960623429641dc7b42a268b1d9c759fa66/src/main/rx/rx.c#L442 (because it needs to be done independently of serial RX or PPM / PWM being used).

Reply:
OK, removed